### ISSUE TITLE
Docker build fail cause by workspace dependency

### ISSUE THREAD

**Issue by KAIYOHUGO** [mdcpp/mdoj#13]
Docker build fail because current Dockerfile only copy member folder.

# TODO List

- [ ] Fix judger
- [ ] Fix backend
- [ ] Move more member dependency to workspace dependency

**Comment by Eason0729** [comment]
I attempted to fix it(**untested**) by copying Cargo.toml(`cp ../Cargo.toml ws-Cargo.toml`) to use it in Dockerfile, which look like this:

```Dockerfile
WORKDIR /complier/proto
COPY backend.proto .
COPY judger.proto .

WORKDIR /compiler
COPY ws-Cargo.toml Cargo.toml

WORKDIR /complier/backend
COPY . .
```

I committed it directly in master as there is no breaking change.

Also, I just forget to add `cp ../Cargo.toml ws-Cargo.toml` in `backend/justfile`.

**Comment by Eason0729** [comment]
```bash
docker build -f /path/to/Dockerfile /path/to/context/folder
```
can be a nice solution

**Comment by KAIYOHUGO** [comment]
Ok, I will fix this, but I mentioned this solution a long time ago.

**Comment by KAIYOHUGO** [comment]
I think it has been fix in fc5636b
